### PR TITLE
ini02 Remove '3D Map Tuning','Racing Features','Board' and add 'FSIO'

### PR DESCRIPTION
**Pull request by andreika-git**

### CODE REVIEW

File: firmware/tunerstudio/rusefi.input [hunk 1]
```diff
@@ -69,10 +69,6 @@ enable2ndByteCanID = false
 ; CONFIG_DEFINITION_END
 	
 
-[SettingContextHelp]
-
-	fuelAlgorithm = "See also http://rusefi.com/fuel"
```
Comment (rusefi) [inline firmware/tunerstudio/rusefi.input]: (I've modified the other PR but same question) why are we removing SettingContextHelp?

File: firmware/tunerstudio/rusefi.input [hunk 2]
```diff
@@ -987,23 +971,13 @@ gaugeCategory = Fuel Data
 		subMenu = std_separator
 		subMenu = std_realtime,    "&Realtime Display",              0
 		subMenu = tChargeSettings, "tCharge Settings"
+		subMenu = std_separator
+		subMenu = smLaunchControl,           "Launch Control"
 		
-	menu = "3D Map Tuning"
-;		subMenu = std_realtime,				"Realtime Display"
-;		subMenu = std_warmup,				"Warmup Wizard"
-		subMenu = fuelTableELMap,			"Fuel Table", 0, {fuelAlgorithm != 3}
```
Comment (rusefi) [inline firmware/tunerstudio/rusefi.input]: what is the plan for 3D map tuning? are we just removing it without moving anywhere?

File: firmware/tunerstudio/rusefi.input (hunk 1 above)
Comment (andreika-git) [reply firmware/tunerstudio/rusefi.input]: Redundancy:
image

The new version is more explanatory:
image

Besides, we've mastered an integrated help system in TunerStudio, and we can add any info or web-links there also:
image

File: firmware/tunerstudio/rusefi.input (hunk 2 above)
Comment (andreika-git) [reply firmware/tunerstudio/rusefi.input]: We're just removing the menu item, but 3D-view is still available as before:
image
If a user likes to work with 3D-maps, there's no need to constantly check this option - it's saved as a preference.

**Pull request merged**